Resources/Prototypes/Entities/Clothing/Head/hats.yml

```diff
@@ -40,17 +44,21 @@
     - ClothMade
     - HamsterWearable
     - WhitelistChameleon
+  - type: DropOnSlip
+    chance: 20
 
 - type: entity
   parent: ClothingHeadBase
   id: ClothingHeadHatCasa
   name: casa
   description: Cone-shaped hat
   components:
-    - type: Sprite
-      sprite: Clothing/Head/Hats/casa.rsi
-    - type: Clothing
-      sprite: Clothing/Head/Hats/casa.rsi
+  - type: Sprite
+    sprite: Clothing/Head/Hats/casa.rsi
+  - type: Clothing
+    sprite: Clothing/Head/Hats/casa.rsi
+  - type: DropOnSlip
+    chance: 20
```

Comment:
```suggestion
    - type: Sprite
      sprite: Clothing/Head/Hats/casa.rsi
    - type: Clothing
      sprite: Clothing/Head/Hats/casa.rsi
    - type: DropOnSlip
      chance: 20
```

Reply:
Was this a mistake? It's spaced like every other entity in this file